# The SelfDetection service returns the master controllerâ€˜ state and the ok flag.

#request constants
string command
---
#response constants
bool ok
string states
string info